lib/customer.rb

```diff
@@ -0,0 +1,56 @@
+require 'csv'
+
+
+class Customer 
+
+    attr_reader :id
+    attr_accessor :email, :address
+
+    def initialize(id,email,address)
+        @id = id
+        @email = email
+        @address = address
+    end
+
+    def self.all 
+        
+        customer_list = []
+        CSV.foreach("data/customers.csv") do |customer|
+        #CSV.foreach("path/to/file.csv", **options) do |row|
+            address = {}
+            id= customer[0].to_i
+            email= customer[1]
+            address[:street] = customer[2]
+            address[:city] = customer[3]
+            address[:state] = customer[4]
+            address[:zip] = customer[5]
+            c = Customer.new(id,email,address)
+            customer_list.push c
+        end
+        return customer_list
+    end
+
+    def self.find(x)
```

Comment:
`x` isn't a great name for this variable. `id`, or `id_to_find` are better, more descriptive alternatives.